src/features/contactverzoek/ContactverzoekFormulier.vue

```diff
@@ -99,41 +97,80 @@
+              @change="setOnderwerp"
+            >
+              <option value="" selected>Geen</option>
+              <option
+                v-for="item in [...data].sort((a, b) => b.id - a.id)"
+                :key="item.id"
+                :value="item.id"
               >
-                <option v-for="item in data" :key="item.id" :value="item.id">
-                  {{ item.naam }}
-                </option>
-              </select>
-            </label>
-            <template v-if="form.contactVerzoekVragenSet">
-              <template
-                v-for="(item, index) in form.contactVerzoekVragenSet
-                  .vraagAntwoord"
-                :key="index"
-              >
-                <label class="utrecht-form-label">
-                  <span> {{ item.vraag }} </span>
-                  <input
-                    class="utrecht-textbox utrecht-textbox--html-input"
-                    type="text"
-                    v-model="item.antwoord"
-                    @input="setActive"
-                  />
-                </label>
-              </template>
+                {{ item.titel }}
+              </option>
+            </select>
+          </label>
+
+          <!-- Dynamic fields based on selected Onderwerp -->
+          <template v-if="form.contactVerzoekVragenSet">
+            <template
+              v-for="(item, index) in form.contactVerzoekVragenSet
+                .vraagAntwoord"
+              :key="index"
+            >
+              <label class="utrecht-form-label">
+                <span>{{ item.description }}</span>
+                <input
+                  v-if="isInputVraag(item)"
+                  class="utrecht-textbox utrecht-textbox--html-input"
+                  type="text"
+                  v-model="item.input"
+                  @input="setActive"
+                />
+                <textarea
+                  v-if="isTextareaVraag(item)"
+                  class="utrecht-textarea"
+                  v-model="item.textarea"
+                  @input="setActive"
+                ></textarea>
+                <select
+                  v-if="isDropdownVraag(item)"
+                  class="utrecht-select"
+                  v-model="item.selectedDropdown"
+                  @input="setActive"
+                >
+                  <option v-for="option in item.options" :key="option">
+                    {{ option }}
+                  </option>
+                </select>
+                <div v-if="isCheckboxVraag(item)">
+                  <label
+                    class="utrecht-checkbox-button"
+                    v-for="(option, optionIndex) in item.options"
+                    :key="option"
+                  >
+                    <input
+                      class="utrecht-checkbox-button"
+                      type="checkbox"
+                      :value="option"
+                      v-model="item.selectedCheckbox[optionIndex]"
```

Comment:
checkbox data binding voorbeeld: 
https://play.vuejs.org/#eNqVU01PAjEQ/StNT5ooe+BG1k2UcJBENOrN9VB2Byh026YfiCH8d6ft7oKJIZJstvPx5s2bTLun91oPth7oiOa2Mly7opSw08o4UsOCeeHIvpSE1Myxq+tkE2LAeSM7j5BqBdUG6hlrwI7Ix2eKH8KBP/zyrGdHx0GjBXOAHiF5zbfFOBEQmRj2+1+U5HDIswAL1VjBpfaOuG8NdyWNwLnalZTwGv01qzZob5nwIT1t3dtG1SA6fEuMiSyJEGwOgiyU6RmKUJlnMREw/+isVvK0c+te0jmWFKHyos4N38BJ56fWvaBzYihC5bFznvWbojfU2UrJBV8O1lZJvDBx+8itGs0FmGftuJJIj+tL+y8pE0J9TWPMGQ83XTzq+SO+tjjSCI0XAxbMFiX1OcfMElxKT95msEO7T+KgXiD6TPIVrBI+aEywBy9rlH2Ci2ofm3D5uVy+28nOgbTdUEFovNARX1J8NeMzox/lDgfD7iHQww+UGiTv

Reply:
![image](https://github.com/Klantinteractie-Servicesysteem/KISS-frontend/assets/132667147/2f3f3b50-dcb8-405a-a9a3-1eb3e2802bc6)

Reply:
als ik op deze manier databind is de value niet altijd uniek, stel iemand heeft een checkbox met dezelfde naam dan wordt die als 1 ref gezien?